dojo/templates/dojo/dashboard.html

```diff
@@ -46,7 +46,7 @@
                         </div>
                     </div>
                 </div>
-                <a href="{% url 'all_findings' %}?duplicate=2&date=2">
+                <a href="{% url 'all_findings' %}?duplicate=2&date=2%o=-date">
```

Comment:
Shouldn't it be `&o=-date`?

Reply:
Yes looks like some encoding artifact while copy and pasting. Will fix